Umbraco-StarterKit/Creating-Content/How-to-Set-Up-Landingpage/index.md

```diff
@@ -0,0 +1,138 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to set up a landingpage
+
+This is a guide designed to help you make a nice landing page, for your website.
+In each step, one feature will be covered. It is important to note that this guide will not cover all options you will have in terms of features.
+
+The Result should look somewhat like this:
+
+![What the page will somewhat look like](images/Frontend-landingpage.gif)
+
+## First Step
+
+- Start by right-clicking the Umbraco Hotel node
+- Choose Create and select page
+- Name your landing page and you are ready to start creating your new landing page
+- In the content section of your page, navigate to the widget section and click "Add content"
+
+## Hero
+
+We will start by adding a Greeting to our page, this will be done using the hero widget, as it allows for having text on top of a background image in a good way.
+
+- Pre heading add part of your heading here this will be in a smaller font size
+- Heading this is the main part of the heading so this will be in big size font
+- The text will be below the heading and in a smaller font size
+- Add a background image
+- Text color black/white depending on the colors of your image
+- Container Width full width
+- Text width 50%
+- Height 70%
+- Overlay 60%
+- Text Align center
+- Vertical text-align top
+- Animate turned on
+- Click Save in the bottom of the page
+- Go to the top of the hero and click the hero tab to minimize it
+
+![Settings for the Hero](images/Heo-Grid.gif)
+
+## Grid
+
+Next, let's add a grid to be able to have a nice frem for how the widgets are structured.
+We will be adding multiple grids to this page because the grid is a great tool with a lot of options.
+
+### The Text and Image Grid
+
+- Click "Add content"
+- Select Grid
+- Select columns
+- Pick Text
+- Column Width 50%
+- Set a name for it in Alias
+- Add the text you want
+- Put boxed to on
+- Click "Add Content"
+- Select Image
+- Name it in alias
+- Column Width 50%
+- Select an image by clicking the +
+- Caption it if you like
+- We won't add link this time
+- Name the grid something in alias
+- Container Width Standard
+- Leave these options off: Remove Column Gutter and Compact
+- Horizontal align left
+- Vertical align top
+- Background image none
+- Background color White
+- Text Color none
+- Animate off
+- Click Save in the bottom of the page
+- Go to the top of the grid and click the grid tab to minimize it
```

Comment:
I've tried splitting this up, so you can see when a new widget starts. Let me know if that doesn't make sense.

```suggestion
- Choose the Text widget
- Column Width: 50%
- Set a name for the item in Alias (this is for display purposes only, and will not show on the frontend)
- Add the text you want
- Toggle *Boxed* to on

- Click "Add Content"
- Select Image
- Name it in alias
- Column Width: 50%
- Select an image by clicking the +

- Name the grid something in alias
- Container Width: Standard
- Leave these options off: Remove Column Gutter and Compact
- Horizontal align: Left
- Vertical align: Top
- Select a light color for the background
- Click Save in the bottom of the page
- Go to the top of the grid and click the grid tab to minimize it
```